IMPLEMENTATION.adoc

```diff
@@ -2,65 +2,70 @@
 
-NOTE: This document reflects the current implementation as of release link:https://github.com/kiali/kiali-operator/tree/v1.69.0[v1.69.0]
+NOTE: This document reflects the current implementation as of release link:https://github.com/kiali/kiali-operator/tree/v1.76.0[v1.76.0]
 
 NOTE: To learn how to set up and run the KO within your own development environment, see link:./DEVELOPING.adoc[DEVELOPING.adoc].
 
 == Ansible
 
-The core of the KO is Ansible. The KO utilizes the link:https://sdk.operatorframework.io/docs/building-operators/ansible/[Ansible Operator SDK] to provide the base Kubernetes operator functionality. The Kiali-specific code is implemented inside Ansible link:https://github.com/kiali/kiali-operator/tree/v1.69.0/playbooks[playbooks] and link:https://github.com/kiali/kiali-operator/tree/v1.69.0/roles[roles].
+The core of the KO is Ansible. The KO utilizes the link:https://sdk.operatorframework.io/docs/building-operators/ansible/[Ansible Operator SDK] to provide the base Kubernetes operator functionality. The Kiali-specific code is implemented inside Ansible link:https://github.com/kiali/kiali-operator/tree/master/playbooks[playbooks] and link:https://github.com/kiali/kiali-operator/tree/master/roles[roles].
 
 NOTE: The KO needs to periodically update its Ansible Operator SDK base image. To see how this is done, just do the same things done previously (see link:https://github.com/kiali/kiali/issues/6220[this issue] and link:https://github.com/kiali/kiali-operator/pull/657/files[this PR] as good examples of what needs to be done).
 
 == Multiple Version Support
 
-The KO can support installing multiple versions of the Kiali server by invoking different versions of the Ansible roles. The versions the KO supports are defined in link:https://github.com/kiali/kiali-operator/blob/v1.69.0/playbooks/default-supported-images.yml[default-supported-images.yml]. For each version supported, there is an link:https://github.com/kiali/kiali-operator/tree/v1.69.0/roles[Ansible role] that the KO executes when it needs to install or remove that Kiali version.
+The KO can support installing multiple versions of the Kiali server and OSSMC by invoking different versions of the Ansible roles. The versions the KO supports are defined in link:https://github.com/kiali/kiali-operator/blob/master/playbooks/kiali-default-supported-images.yml[kiali-default-supported-images.yml] and (for OSSMC) in link:https://github.com/kiali/kiali-operator/blob/master/playbooks/ossmconsole-default-supported-images.yml[ossmconsole-default-supported-images.yml]. For each version supported, there is an link:https://github.com/kiali/kiali-operator/tree/master/roles[Ansible role] that the KO executes when it needs to install or remove that Kiali or OSSMC version.
```

Comment:
The Kiali and OSSMC versions must be the same, right, they are linked?

Reply:
yeah, today they need to be linked.